Content.Server/_Floof/ModifyUndies/ModifyUndiesSystem.cs

```diff
@@ -62,22 +60,7 @@ private void AddModifyUndiesVerb(EntityUid uid, ModifyUndiesComponent component,
             // check if the Bodypart is in the component's BodyPartTargets
             if (!component.BodyPartTargets.Contains(mProt.BodyPart))
                 continue;
-                
-            // Skip genital markings based on consent
-            if (mProt.BodyPart == HumanoidVisualLayers.Genital)
-            {
-                // If user and target are the same person, they can always interact with their own markings
-                if (args.User != args.Target)
-                {
-                    // For other players, only check the target's consent setting
-                    var hasTargetConsent = _consentSystem.HasConsent(args.Target, GenitalMarkingsConsent);
-                    if (!hasTargetConsent)
-                    {
-                        continue;
-                    }
-                }
-            }
-            
+
```

Comment:
I get why this change is here, though I feel like it should be reworked. An important part of genitals is being able to hide/show individual parts, mainly since we dont have logic in place to dynamically hide them when they should, reasonably, be hidden (undies, clothes, etc). It can be a pretty big issue for larger genitals, since they stick out a lot.

What I'd suggest is to have genitals use a different Category, as that'll put them all in a separate dropdown list. Then, use another consent pref that is checked if the user != target, something like "Allow others to adjust genital visibility".

Reply:
Can you give me an example to test this? I can probably add logic to hide certain genitals when under underwear.

Reply:
- also made it so the character editor's dummy doll's genitals are always visible, regardless of consent. I wanted to make it always visible on the player's mob, but I couldnt find a quick way to figure out if the component being checked was owned by the mob controlled by the client.